File: week-3-algorithms/lecture-3/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

typedef struct
{
    string name;
    string number;
}
person;

int main(void)
{
    person people[2];

    people[0].name = "Zidan";
    people[0].number = "+62-823-456-789-10";

    people[1].name = "Abraham";
    people[1].number = "+62-123-456-789-21";

    string name = get_string("Name: ");

    for (int i = 0; i < 2; i++)
    {
        if (strcmp(people[i].name, name) == 0)
        {
            printf("Found %s\n", people[i].number);
            return 0;
        }
    }

    printf("Not Found\n");
    return 1;
}

File: week-6-python/lecture/phonebook.py
people = {
    "zidan": "1234567891",
    "abra": "9876543219",
    "john": "123454321",
}

name = input("Name: ").lower()

if name in people:
    print(f"Number: {people[name]}")
else:
    print("Not Found")
